utils/calc-stack-height/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "calc-stack-height"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
```suggestion
```

Reply:
Why do we remove these lines?

Reply:
Why no? It's template-informational comments